build.py

```diff
@@ -0,0 +1,143 @@
+#   -*- coding: utf-8 -*-
+import logging
+from pathlib import Path
+
+from pybuilder.core import use_plugin, init, before, after, Author
+from pybuilder.errors import BuildFailedException
+
+use_plugin("python.core")
+use_plugin("python.flake8")
+use_plugin("python.distutils")
+# use_plugin("python.unittest")
+# use_plugin("python.coverage")
+# https://github.com/AlexeySanko/pybuilder_pytest
+# use_plugin('pypi:pybuilder_pytest')
+# https://github.com/AlexeySanko/pybuilder_pytest_coverage
+# use_plugin('pypi:pybuilder_pytest_coverage')
```

Comment:
Could we remove this commented code?

Reply:
I would like to keep them there as it was a reminder for the next steps: coverage & pytest within pybuilder.